charts/registry-scanner/values.yaml

```diff
@@ -67,9 +67,9 @@ config:
     # List of regular expressions. Images matching any of these expressions are excluded when scanning.
     exclude: []
     # Exclude images with creation date older than specified number of days.
-    maxAgeDays:
+    maxAgeDays: 0
```

Comment:
I would suggest commenting here and in the README.md that the value "0" means unlimited or no filter. Otherwise it might not be straightforward for the user.